mappings/net/minecraft/entity/mob/HoglinEntity.mapping

```diff
@@ -3,9 +3,24 @@ CLASS net/minecraft/class_4760 net/minecraft/entity/mob/HoglinEntity
 	FIELD field_22366 MEMORY_MODULE_TYPES Lcom/google/common/collect/ImmutableList;
 	FIELD field_22367 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_22368 movementCooldownTicks I
+	FIELD field_23735 timeInOverworld I
+	FIELD field_23736 cannotBeHunted Z
+	FIELD field_23737 BABY Lnet/minecraft/class_2940;
 	METHOD method_24349 canSpawn (Lnet/minecraft/class_1299;Lnet/minecraft/class_1936;Lnet/minecraft/class_3730;Lnet/minecraft/class_2338;Ljava/util/Random;)Z
+		ARG 0 entityType
 		ARG 1 world
+		ARG 2 spawnType
 		ARG 3 pos
 	METHOD method_24658 isAdult ()Z
 	METHOD method_24659 playFightSound ()V
 	METHOD method_24661 playRetreatSound ()V
+	METHOD method_26942 zombify (Lnet/minecraft/class_3218;)V
+		ARG 1 word
+	METHOD method_26944 canConvert ()Z
+	METHOD method_26945 getCanBeHunted ()Z
```

Comment:
Hum, not a good name for a boolean getter

Reply:
Should just be `canBeHunted`

Reply:
```suggestion
	METHOD method_26945 canBeHunted ()Z
```